pkg/k8s/controllers/telemetryservice_controller.go

```diff
@@ -68,3 +97,78 @@ func (r *TelemetryServiceReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&v1alpha1.TelemetryService{}).
 		Complete(r)
 }
+
+func CreateTelemetryServiceService(ctx context.Context, r *TelemetryServiceReconciler, ts *v1alpha1.TelemetryService, req ctrl.Request) error {
+	rlog := log.FromContext(ctx)
+	svc := &corev1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      tsNamespacedName.Name,
+			Namespace: tsNamespacedName.Namespace,
+		},
+		Spec: corev1.ServiceSpec{
+			Selector: map[string]string{"app": tsNamespacedName.Name},
+			Ports: []corev1.ServicePort{
+				{
+					Port:       80,
+					Protocol:   corev1.ProtocolTCP,
+					TargetPort: intstr.IntOrString{IntVal: 8080},
+				},
+			},
+			Type: corev1.ServiceTypeLoadBalancer,
+		},
+	}
+	rlog.Info("Start Creating service")
```

Comment:
"start creating TelemetryService service"
error msg could follow: 
1. use downcase
2. keep uppercase unless its an specific terminology, here "TelemetryService" is a specific terminology.

Reply:
Thanks a lot for your advice!